frontend/src/components/buttons/CreateButtonMobile.test.ts

```diff
@@ -53,13 +65,80 @@ describe('CreateButton', () => {
       wrapper = Wrapper()
     })
     describe('enter room', () => {
-      beforeEach(async () => {
-        await wrapper.find('#create-button-mobile').trigger('click')
-        await wrapper.find('.button-list-mobile button.new-table-button').trigger('click')
+      describe('apollo with success', () => {
+        beforeEach(async () => {
+          vi.clearAllMocks()
+          joinMyRoomMutationMock.mockResolvedValue({
+            data: {
+              joinMyRoom: 'http://link-to-my.room',
+            },
+          })
+          await wrapper.find('#create-button-mobile').trigger('click')
+          await wrapper.find('button.new-table-button').trigger('click')
+        })
+
+        it('calls the api', () => {
+          expect(joinMyRoomMutationMock).toBeCalled()
+        })
+
+        it('updates the store', () => {
+          expect(activeRoomStore.activeRoom).toBe('http://link-to-my.room')
+        })
+
+        it.skip('navigates to room page', () => {
+          expect(navigate).toBeCalledWith('/room/')
+        })
       })
 
-      it('opens room page', () => {
-        expect(navigate).toHaveBeenCalledWith('/room/')
+      describe('apollo with no data', () => {
+        beforeEach(async () => {
+          activeRoomStore.setActiveRoom(null)
+          vi.clearAllMocks()
+          joinMyRoomMutationMock.mockResolvedValue({
```

Comment:
What about backend errors here:
```suggestion
          joinMyRoomMutationMock.mockResolvedValue({
            error: // ??
```

Reply:
Good question. I still don't know enough about error handling with Graphql to have an answer at hand, but will investigate...